### PR TITLE
Markets in dispute

### PR DESCRIPTION
**Pull request by JohnDanz**
https://github.com/AugurProject/augur/issues/2925

### CODE REVIEW

**Review by phoebemirman — CHANGES_REQUESTED** [review]
seeing markets show under both tabs

File: packages/augur-ui/src/modules/reporting/components/reporting-report/reporting-report.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React, { Component } from "react";
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/components/reporting-report/reporting-report.tsx]: is this file needed?

File: packages/augur-ui/src/modules/reporting/components/reporting-dispute-markets/reporting-dispute-markets.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
+import { Helmet } from "react-helmet";
+
+export default class ReportingDisputeMarkets extends Component {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/components/reporting-dispute-markets/reporting-dispute-markets.tsx]: is this file needed?

File: packages/augur-ui/src/modules/reporting/disputing.tsx [hunk 3]
```diff
@@ -31,8 +29,9 @@ export default class Disputing extends Component<DisputingProps> {
 
   render() {
     return (
-      <section>  
-        <ParticipationTokensView />
+      <section>
+        {/* <ParticipationTokensView /> */}
+        <MarketsInDispute />
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/disputing.tsx:34]: why is this file pulling in <MarketsInDisput />?

**Review by bthaile — COMMENTED** [review]
Some questions why some files exist.

**Review by phoebemirman — CHANGES_REQUESTED** [review]
Screen Shot 2019-09-10 at 9 52 38 AM
getting this error when not signed in

File: packages/augur-ui/src/modules/reporting/components/markets-in-dispute.tsx [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+  sortBy: string;
+}
+
+const sortByOptions = [
+  {
+    label: 'Amount REP Staked',
+    value: 'repStaked',
+    comp(marketA, marketB) {
+      return (
+        parseFloat(marketB.disputeInfo.stakeCompletedTotal) -
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/reporting/components/markets-in-dispute.tsx]: should you use big number here?

**Review by phoebemirman — APPROVED** [review]
looks good, small comment

**Review by bthaile — APPROVED** [review]
comments will be addressed in next PR

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🔮

 This link will always point to this PR's latest changes: https://peek.run/o9ph6p3
